src/Mod/Arch/ArchAxis.py

```diff
@@ -224,7 +226,7 @@ def __init__(self,vobj):
 
     def setProperties(self,vobj):
 
-        ts = FreeCAD.ParamGet("User parameter:BaseApp/Preferences/Mod/Draft").GetFloat("textheight",350)
+        ts = params.get_param("textheight")
```

Comment:
Not sure why we specifically provided a default here, but might be good to check?